ert_storage/graphql/ensembles.py

```diff
@@ -17,8 +18,21 @@ class Meta:
 
     child_ensembles = gr.List(lambda: Ensemble)
     parent_ensemble = gr.Field(lambda: Ensemble)
-    responses = gr.Field("ert_storage.graphql.responses.Response")
+    response_names = gr.List(gr.String)
+    responses = gr.List("ert_storage.graphql.responses.Response")
+    parameter_names = gr.List(gr.String)
```

Comment:
I was thinking a bit, and I think having a `uniqueParameters` field rather than a `parameterNames`. It's more flexible, and would allow us to do eg:
```
uniqueParameters {
    name
    prior
}
```

Reply:
I've removed `parameters` endpoint as we will use the `inputs` instead. We can discuss the `uniqueParameters` to get the priors though.

Reply:
> I was thinking a bit, and I think having a `uniqueParameters` field rather than a `parameterNames`. It's more flexible, and would allow us to do eg:
> 
> ```
> uniqueParameters {
>     name
>     prior
> }
> ```

I would then add the same for responses:
```python
uniqueResponses {
    name
    observations
}
```